INSTALL/install.sh

```diff
@@ -124,3 +124,9 @@ mkdir -p /var/log/reform
 cp ./conf/supervisor*ini /etc/supervisord.d/
 systemctl start supervisord
 systemctl enable supervisord
+
+
+# Create the following cron job
+crontab -e
+# add following command to the files, and save it.
+0 1 * * * /bin/bash /home/reform/reformWeb/cleanup.sh
```

Comment:
@YuWei-CH I updated the cron time to be daily at 1am rather than hourly. Also cleanup.sh is a bash script so I changed the python call to bash.

Reply:
Thank you Professor.